src/js/components/SearchAllBox.js

```diff
@@ -284,18 +296,16 @@ export default class SearchAllBox extends Component {
                 let today = new Date();
                 let election_date = new Date(election_day + " 0:00:00");
                 let past_election = today > election_date ? " IN PAST" : "UPCOMING ELECTION";
-                console.log("Election election_date ", election_date);
                 return <Link key={one_result.local_id}
                              data-idx={idx}
-                             to={this.links[idx]}
                              onMouseOver={this.onSearchResultMouseOver}
                              className="search-container__links"
-                             onClick={this.onSearchResultClick}>
+                             onClick={this.onSearchElectionResultClick.bind(this, one_result.google_civic_election_id)}>
```

Comment:
Hi @jainanisha90 I think instead of passing in "this" we will want to bind this to the function in the constructor -- I can show you that tomorrow.